### PR TITLE
A note on evidence

### LINKED ISSUE #85: Add two-pager on evidence about programming language usability
http://quorumlanguage.com/evidence/Evidence.pdf

### PR DESCRIPTION
**Pull request by gvwilson**
Closes #85.

### CODE REVIEW

File: instructors.md [hunk 1]
```diff
@@ -199,6 +199,41 @@ A typical schedule for a two-day course is:
...
+We believe our core topics are the absolute minimum that researchers need to know
+in order to work efficiently and reproducibly.
+More importantly,
+we don't know what we could take out to make space for something else.
+
+One thing we *do* know is that we do not wish to become embroiled in debates
+over the relative merits of different languages or operating systems.
+No one has ever demonstrated that R programmers are more productive than Python programmers,
+and proficient users of Windows seem just as productive as equally-proficient users of Unix.
+If a learner asserts that their favorite tool is better than alternatives,
```
Comment (davis68) [inline instructors.md]: also, perhaps suggest distinguishing between "better" and "better for me" to the assertor

**Comment by davis68** [comment]
Have you committed the paper somewhere?  I can't find it in any of the branches or in your fork.

**Comment by gvwilson** [comment]
My mistake - I thought I had committed this. It's in the PR now.

**Comment by davis68** [comment]
Got it.  Do you object to *also* linking the [original ACM page](https://dl.acm.org/citation.cfm?id=2534973) (it's behind a paywall, I believe)?

**Comment by gvwilson** [comment]
Sure, I'll add the link for those privileged enough to be able to access 
it :-)
G

On 2016-03-21 11:02 AM, Neal Davis wrote:
>
> Got it. Do you object to /also/ linking the original ACM page 
> <https://dl.acm.org/citation.cfm?id=2534973> (it's behind a paywall, I 
> believe)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/instructor-training/pull/136#issuecomment-199328140>
>

-- 
Dr Greg Wilson
Director of Instructor Training
Software Carpentry Foundation

**Comment by gvwilson** [comment]
I've added a link to the paper, but the two-pager isn't the same thing.

**Comment by davis68** [comment]
Got it.  This all looks good to me then.

**Pull request merged**